parley/src/layout/run.rs

```diff
@@ -112,7 +112,7 @@ impl<'a, B: Brush> Run<'a, B> {
     }
 
     /// Returns an iterator over the clusters in logical order.
-    pub fn clusters(&'a self) -> impl Iterator<Item = Cluster<'a, B>> + Clone {
+    pub fn clusters(&'a self) -> impl Iterator<Item = Cluster<'a, B>> + 'a + Clone {
```

Comment:
Something something this should probably use the captures trick, but it's also underdocumented and everyone gets this wrong, and we'll be moving to 1.85 in the reasonably near future anyway.